src/main/java/org/cardanofoundation/cfexploreraggregator/addresstxcount/processor/AddressTxCountProcessor.java

```diff
@@ -70,13 +70,13 @@ public void handleByronEvent(ByronMainBlockEvent event) {
     }
 
     private void processTransaction(long slot, String address) {
-        try {
+        if(address.startsWith("addr") || address.startsWith("stake")) {
             Address addr = new Address(address);
             if (addr.getAddressType() == AddressType.Base) {
                 String stakeAddress = AddressProvider.getStakeAddress(addr).getAddress();
                 addToMap(slot, stakeAddress);
             }
```

Comment:
Couple of questions:
1. are we ever gonna see a `stake1...` being passed down here?
2. @satran004 is there a better way via CCL to classify addresses?

Reply:
I was chatting with Satya and he suggested to two options. First check with `startsWith("addr")` or check if `new ByronAddress(addr)` throws an exception. And I don't want to do control flow by exception handling. So I decided to use the first option. Might be useful to put it to CCL to check the Era of an address. 

Regarding `stake1` you are probably right, just added it here for completeness, but I can remove it.